### PR TITLE
Update logging example

### PR DESCRIPTION
**Pull request by satnam6502**
This PR trims down the Makefile to just the rules needed to create and tear down the synthetic logging sources and the README.md is updated to point to the relevant logging getting started guide.
@a-robinson

### CODE REVIEW

**Comment by satnam6502** [comment]
@caesarxuchao

File: examples/logging-demo/Makefile [hunk 1]
```diff
@@ -1,34 +1,26 @@
 # Makefile for launching syntheitc logging sources (any platform)
 # and for reporting the forwarding rules for the
 # Elasticsearch and Kibana pods for the GCE platform.
+# For examples of how to observe the ingested logs please
+# see thea appropriate getting started guide e.g.
```
Comment (a-robinson) [inline examples/logging-demo/Makefile]: s/thea/the

File: examples/logging-demo/Makefile [hunk 2]
```diff
@@ -1,34 +1,26 @@
...
+# Google Cloud Logging: https://github.com/GoogleCloudPlatform/kubernetes/blob/master/docs/getting-started-guides/logging.md
+# GCE with Elasticsearch and Kibana: https://github.com/GoogleCloudPlatform/kubernetes/blob/master/docs/getting-started-guides/logging-elasticsearch.md
```
Comment (a-robinson) [inline examples/logging-demo/Makefile]: Do you really need to qualify this as being for GCE? It's very generic to different cloud providers other than the output of the cluster-up command.

**Comment by satnam6502** [comment]
Thanks. Fixed the typo and also spelling of synthetic.

File: examples/logging-demo/Makefile (hunk 2 above)
Comment (satnam6502) [inline examples/logging-demo/Makefile]: Reworded. Thanks.

**Comment by a-robinson** [comment]
LGTM

**Pull request merged**